packages/shopify-codemod/transforms/split-if-assignments.js

```diff
@@ -0,0 +1,13 @@
+export default function splitIfAssignments({source}, {jscodeshift: j}, {printOptions = {}}) {
+  return j(source)
+    .find(j.IfStatement, {
+      test: {
+        type: j.AssignmentExpression.name,
+      },
+    }).forEach((ifPath) => {
```

Comment:
do we need to name it `ifPath`? I think that `path` is fine since the `find` is explicitly on `j.IfStatement`s